app/ts/components/simulationExplaining/SimulationSummary.tsx

```diff
@@ -688,13 +686,26 @@ export function ExtraDetailsTransactionCard({ transaction }: ExtraDetailsTransac
 		{ !showSummary ? <></> : <>
 			<div class = 'card-content'>
 				<div class = 'container' style = 'margin-bottom: 10px;'>
-					<span class = 'log-table' style = 'justify-content: center; column-gap: 5px; grid-template-columns: auto auto'>
-						<CellElement text = 'Transaction Type: '/>
+					<span class = 'log-table' style = 'justify-content: center; column-gap: 5px; row-gap: 5px; grid-template-columns: auto auto'>
+						<CellElement text = 'Transaction type: '/>
 						<CellElement text =  { transaction.type }/>
-						<CellElement text = 'Nonce: '/>
-						<CellElement text = { transaction.nonce.toString(10) }/>
+						<CellElement text = 'From: '/>
+						<CellElement text = { <SmallAddress addressBookEntry = { transaction.from } renameAddressCallBack = { renameAddressCallBack } textColor = { 'var(--subtitle-text-color)' } /> } />
+						<CellElement text = 'To: '/>
+						<CellElement text = { transaction.to === undefined ? 'No receiving Address' : <SmallAddress addressBookEntry = { transaction.to } renameAddressCallBack = { renameAddressCallBack } textColor = { 'var(--subtitle-text-color)' }/> } />
+						<CellElement text = 'Value: '/>
+						<CellElement text = { <Ether amount = { transaction.value } useFullTokenName = { true } chain = { transaction.chainId } textColor = { 'var(--subtitle-text-color)' }/> } />
+						<CellElement text = 'Gas used: '/>
+						<CellElement text = { `${ gasSpent.toString(10) } gas (${ Number(gasSpent * 10000n / transaction.gas) / 100 }%)` }/>
 						<CellElement text = 'Gas limit: '/>
 						<CellElement text = { `${ transaction.gas.toString(10) } gas` }/>
+						<CellElement text = 'Nonce: '/>
+						<CellElement text = { transaction.nonce.toString(10) }/>
+						<CellElement text = 'Chain: '/>
+						<CellElement text = { getChainName(BigInt(transaction.chainId)) }/>
+						<CellElement text = 'Unsigned transaction hash: '/>
+						<CellElement text = { bytes32String(transaction.hash) }/>
```

Comment:
That transaction hash is taking up a lot of space and isn't necessary for anything.  It can be calculated from the rest of the transaction details and I don't think any tools use it for anything.

Perhaps keep it, but greatly constrain how much space it takes up and let people click to copy it if they want the whole thing, or hover tooltip for whole thing?

Reply:
I imagine that if you want to sign that transaction offchain, that's the only information you need? Sure Interceptor could lie about it

Reply:
well there's space for it, and if there's not it gets ellipsed

Reply:
The problem is that it causes the rest of the fields to be separated from their labels by a lot.  What if we right justify the left column and left justify the right column?  How does that look?